app/renderer/components/tabs/content/tabTitle.js

```diff
@@ -70,28 +62,28 @@ class TabTitle extends React.Component {
 module.exports = ReduxComponent.connect(TabTitle)
 
 const styles = StyleSheet.create({
-  tabTitle: {
+  tab__title: {
+    boxSizing: 'border-box',
     display: 'flex',
-    flex: '1',
+    flex: 1,
     userSelect: 'none',
-    boxSizing: 'border-box',
     fontSize: globalStyles.fontSize.tabTitle,
-    overflow: 'hidden',
-    whiteSpace: 'nowrap',
     lineHeight: '1.6',
-    padding: globalStyles.spacing.defaultTabPadding,
-    color: 'transparent',
-    WebkitBackgroundClip: 'text',
-    // prevents the title from being the target of mouse events.
-    pointerEvents: 'none'
+    minWidth: 0, // see https://stackoverflow.com/a/36247448/4902448
+    marginLeft: '4px',
+    overflow: 'hidden'
   },
 
-  enforceFontVisibility: {
-    fontWeight: '600'
+  tab__title_bold: {
```

Comment:
I am not sure if this would be a proper name as we have `tab__title_windows` below which has more bold font-weight. How about renaming it to `tab__title_isDarwin`?

Reply:
Confirmed this was addressed on https://github.com/brave/browser-laptop/pull/10691/files#diff-6c5463a6f60c16525fb939516ad54125R53

On the commit: https://github.com/brave/browser-laptop/commit/d97a56b311bc994f48383dc5c7b0929b0c0bf57a#diff-6c5463a6f60c16525fb939516ad54125R53